packages/core/src/components/icon-button/icon-button.tsx

```diff
@@ -74,6 +74,11 @@ export class IconButton implements Button {
    */
   @Prop() type: 'button' | 'submit' = 'button';
 
+  /**
+   * Enable dropdown button triangle
+   */
+  @Prop() dropdownTriangle = false;
+
```

Comment:
We should not add the `dropdown-triangle` as feature flag to icon button.

Please implement the dropdown icon in dropdown button itself.